packages/editor-ui/src/components/Node.vue

```diff
@@ -16,6 +16,11 @@
 				</n8n-tooltip>
 			</div>
 
+			<n8n-tooltip :placement="'top'" :manual="true" :open-delay="2500" :value="showWebhookNodeTooltip">
```

Comment:
remove `open-delay` if it's not doing anything
set `:placement="top"` you don't need the extra quotes

Reply:
This is removed.